great-idea/services.html

```diff
@@ -14,94 +14,100 @@
+          Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+      </p>
+    </div>
+
+
+<div class="row">
+  <div class="item">
+    <h2>Digital Design</h2>
+    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+        eleifend. Mauris euismod facilisis iaculis.</p>
+        <button>Learn More</button>
+  </div>
+  <div class="item">
+    <h2>UX / UI</h2>
+    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+        eleifend. Mauris euismod facilisis iaculis.</p>
+        <button>Learn More</button>
+  </div>
+</div>
+
+<div class="row">
+  <div class="item">
+  <h2>Digital Marketing</h2>
+  <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+      eleifend. Mauris euismod facilisis iaculis.</p> 
+      <button>Learn More</button>
+</div>
+  <div class="item">
+    <h2>Web Development</h2>
+    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+        eleifend. Mauris euismod facilisis iaculis.</p>
+        <button>Learn more</button>
+  </div>
+</div>
+
+<div class="row">
+    <div class="item">
+    <h2>iOS Development</h2>
+    <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+        eleifend. Mauris euismod facilisis iaculis.</p> 
+        <button>Learn More</button>
+  </div>
+    <div class="item">
+      <h2>Android Development</h2>
+      <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+          eleifend. Mauris euismod facilisis iaculis.</p>
+          <button>Learn more</button>
+    </div>
+  </div>
+
+<section class="services facts">
```

Comment:
```suggestion
<section class="services facts">
<div class="some facts">
<h2>Some Facts About Our Services</h2>
<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
    eleifend. Mauris euismod facilisis iaculis.</p>
<ul>
  <li>Awesome thing</li>
  <li>Amazing thing</li>
  <li>Cool thing</li>
  <li>Great thing</li>
</ul>
</div>
<img class="services-info-img" src="img/services-info.png" alt="one of our employees hard at work">
</section>
```

Reply:
same here the container needs a display flex and the children need their own containers with width sizes